components/xsd-fu/templates-java/OMEXMLModelEnum.template

```diff
@@ -61,7 +61,30 @@ import ome.xml.model.primitives.*;
 
 public enum ${klass.langType} implements Enumeration
 {
-  ${", ".join(['%s("%s")' % (enum_value_name(value, klass.isUnitsEnumeration), value) for value in klass.possibleValues])};
+{% for value in klass.possibleValues %}\
+{% if klass.enumProperties is not None and value in klass.enumProperties and klass.enumProperties[value].get('enum', None) is not None %}\
+{% if value == klass.possibleValues[-1] %}\
+  ${klass.enumProperties[value].enum}("${value}") /** ${klass.enumDocumentation[value]} */;
```

Comment:
I thought this was valid javadoc syntax for enum values?  No matter though, I've pushed a change to move the comments onto the preceding line.